packages/commonwealth/server/routes/addEditors.ts

```diff
@@ -1,6 +1,6 @@
 import { Request, Response, NextFunction } from 'express';
 import { Op } from 'sequelize';
-import { findOneRole } from 'server/util/roles';
+import { findOneRole } from 'commonwealth/server/util/roles';
```

Comment:
This should be a relative path (i.e. `../util/roles`) -- we shouldn't ever use absolute paths on the server.

Reply:
yes!